### DISCUSSION TITLE
Refetch Status

### DISCUSSION THREAD

**Discussion by xavierakram** [TanStack/query#2003] [Q&A]
Firstly, big love for the React-Query project, this thing is a god send.

I have a question about determining the status of a refetch();

when you setup a query using useQuery, you get access to a status value which holds, the loading, success etc.

I was wondering if there was something similar for refetch?

**Comment by TkDodo** [comment]
Not sure what you mean exactly. Are you talking about the `refetch` method returned by useQuery?

**Reply by xavierakram** [reply]
Yes exactly I'm wondering if refresh from useQuery maintains its own status.

**Reply by TkDodo** [reply]
It does not - it belongs to the query and thus changes the status of the query.
